examples/typescript/.sauce/config.yml

```
@@ -0,0 +1,28 @@
+apiVersion: v1alpha
+kind: puppeteer
+sauce:
+  region: us-west-1
+  concurrency: 2 # Controls how many suites are executed at the same time.
+  metadata:
+    name: saucectl puppeteer example
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+defaults:
+  mode: docker
+docker:
+  # Affects how test files are transferred to the docker container when using the docker mode (--test-env docker).
+  fileTransfer: copy # Choose between mount|copy.
+# Controls what files are available in the context of a test run (unless explicitly excluded by .sauceignore).
+rootDir: ./
+puppeteer:
+  version: 8.0.0 # See https://docs.saucelabs.com/testrunner-toolkit for a list of supported versions.
+suites:
+  - name: "chrome"
+    testMatch: ["**/*.test.js"] # jest native glob support.
```

I guess this should be this way:
```suggestion
    testMatch: ["**/*.test.ts"] # jest native glob support.
```

---

I couldn't find information that puppeteer supports typescript. Users can use typescript for writing tests, but after that, they need to compile and use javascript implementation.

---

Jest supports typescript, so support should be simple if we make jest support it:
https://jestjs.io/docs/getting-started#using-typescript

Currently there is no typescript support in puppeteer image.

---

I am not sure it's the scope of this task. We can create a separate ticket for that.